deploy/arax/scripts/download_database.sh

```diff
@@ -81,7 +81,7 @@ sftp -i $sftp_key -o StrictHostKeyChecking=no -pr ${sftp_url}/md5_sums/ $md5_sum
 printf '\n'
 
 # this is tricky, we download all MD5 info, some are needed, some are not
-# we need to loop through all downloaded MD5s, remove others but only keep the needed ones, which is from the configv2.json
+# we need to loop through all downloaded MD5s, remove others but only keep the needed ones, which is from the config_secrets.json
```

Comment:
Note that `config_secrets.json` does not contain the databases that need to be downloaded; `RTX/code/config_dbs.json` does.

Reply:
This is fine.

Reply:
Ok, just clarifying since multiple comments above refer to `config_secrets.json` as containing the DBs.